src/main/webapp/components/input/intrigue-radio-button.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import FormLabel from '@material-ui/core/FormLabel'
+import ButtonGroup from '@material-ui/core/ButtonGroup'
+import Button from '@material-ui/core/Button'
+
+export const RadioButton = props => {
+  const { label, buttonText, defaultButton, onChange } = props
+  const [selectedButton, setSelectedButton] = React.useState(defaultButton)
```

Comment:
I think this component should be stateless and have the selected button derived from props

I would also recommend structuring the `buttonText` objects as an array of : `{ label, value }` so you don't need to worry about handling indices